ami_assets/etc/xrd/xrd-vrouter-build-ami.sh

```diff
@@ -68,29 +70,24 @@ install_ami_assets() {
 }
 
 build_igb_uio() {
-    local DPDK_FILE
-
-    DPDK_FILE=$(basename "$DPDK_SRC")
-
     # Want whitespace splitting here for multiple packages.
     # shellcheck disable=SC2086
     sudo yum install -y $DPDK_BUILD_PACKAGES
     sudo yum groupinstall -y "$DPDK_BUILD_GROUP"
 
-    wget "$DPDK_SRC"
-    tar xf "$DPDK_FILE"
-    cd "$DPDK_DIR"
+    git clone "$DPDK_KMODS_REPO" "$DPDK_KMODS_DIR"
```

Comment:
`https://git.dpdk.org/dpdk-kmods/snapshot/dpdk-kmods-e721c733cd24206399bebb8f0751b0387c4c1595.tar.gz` also works here if you want to avoid the git clone.

I'm not sure if there are any guarantees on the snapshots being available though - but other Linux distros generally use the snapshots rather than git.

Reply:
Looks good to me, will update.